src/classes/EscalationTrigger.cls

```diff
@@ -193,6 +194,24 @@ public with sharing class EscalationTrigger extends BaseTrigger {
 	}
 
 	/**
+	* Assign default owner to an escalation
+	*/
+	@TestVisible
+	private void assignDefaultOwner() {
+		if (!this.isInsert) {
+			return;
+		}
+
+		rh_escal__EscalationRouter__c er = rh_escal__EscalationRouter__c.getValues(DEFAULT_ESCAL_ROUTER);
```

Comment:
@pcon may be earlier I was not clear that for escalation we don't have a recordType, so we can not use 'rh_escal__EscalationRouter__c router = EscalationRouterUtils.getRouterForRecordType(escalation.RecordTypeId);' becuase this will throw a compilation error as 'Invalid field RecordTypeId for SObject rh_escal__Escalation__c'.

This is the reason because of which I asked for using a DEFAULT router rather than going for recordTypes. 
Please suggest....